server/src/interfaces.ts

```diff
@@ -177,6 +177,14 @@ export interface Filter {
   value: any;
 }
 
+/**
+ * OrderRule Interface that contains order rules for sorting.
```

Comment:
add comment about the order if descending not specified

Reply:
@ifhuang okay!

Reply:
@ifhuang Added the comment in datastore.ts in the JSDoc description for `getAll` function [here](https://github.com/googleinterns/gpay-group-buy/pull/160/commits/6f8cdbac1fd29909eb449142563514778c9cb2e3). This is because the default direction is only specified by Datastore's `query.order()` function. But this interface here doesn't specify the default direction.